e2e/test/scenarios/filters-reproductions/filters-reproductions.cy.spec.js

```diff
@@ -1200,6 +1206,62 @@ describe("issue 40622", () => {
   }
 });
 
+describe("45252", { tags: "@external" }, () => {
+  beforeEach(() => {
+    resetTestTable({ type: "postgres", table: "many_data_types" });
+    restore("postgres-writable");
+    cy.signInAsAdmin();
+    resyncDatabase({ dbId: WRITABLE_DB_ID, tableName: "many_data_types" });
+    cy.intercept("POST", "/api/dataset").as("dataset");
+  });
+
+  it("should allow using is-null and not-null operators with unknown data types (metabase#45252)", () => {
+    startNewQuestion();
+
+    cy.log("filter picker - new filter");
+    entityPickerModal().within(() => {
+      entityPickerModalTab("Tables").click();
+      cy.findByText("Writable Postgres12").click();
+      cy.findByText("Many Data Types").click();
+    });
+    getNotebookStep("filter")
+      .findByText("Add filters to narrow your answer")
+      .click();
+    popover().within(() => {
+      cy.findByText("Binary").scrollIntoView().click();
+      cy.findByLabelText("Is empty").click();
+      cy.button("Add filter").click();
+    });
+    visualize();
+    cy.wait("@dataset");
```

Comment:
`visualize` already does `cy.wait("@dataset");`

```suggestion
    visualize();
```